client/src/components/ManageAccount/ManageAccountModals.js

```diff
@@ -167,12 +167,14 @@ export const GoogleDriveModal = ({ active, toggle }) => {
             limited to mimeType, name, id, modifiedTime, fileExtension, size,
             and parents.
           </p>
-          <Button
-            href="/accounts/applications/googledrive/initialize/"
-            className="manage-account-submit-button"
-          >
-            <span>Agree and Connect to Google Drive</span>
-          </Button>
+          <div className="google-drive-submit-button-container">
+            <Button
+              href="/accounts/applications/googledrive/initialize/"
+              className="manage-account-submit-button"
+            >
+              <span>Agree and Connect to Google Drive</span>
+            </Button>
+          </div>
```

Comment:
[In light of this looking good in prod at the moment,](https://github.com/TACC/Core-Portal/pull/354/files#r587960386) I suggest we roll back this change:
```suggestion
          <Button
            href="/accounts/applications/googledrive/initialize/"
            className="manage-account-submit-button"
          >
            <span>Agree and Connect to Google Drive</span>
          </Button>
```